docs/live-sim.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Live Simulation Mode
+---
+
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Live Simulation Mode
+
+Forget screen sharing, there’s live simulation mode! Let’s say you’re working on the application logic and want a second opinion or to share your machine with your team mates, live simulation has made this process as easy as sharing a link. It is not necessary to take screenshots anymore because this feature will let all users with the specially generated link to access and discuss in real time the process of your machine.
+
+In order to use this collaborative feature, the user initiating the live simulation must be signed in. When looking at the Stately Studio, you’re able to switch between two views: Design mode and Simulate mode.
+
+:::studio
+
+Live simulation is a Pro feature of the Studio. [Check out the features on our Pro plan](/studio-pro-plan.mdx) or [upgrade to try the Pro plan with a 30-day free trial](https://stately.ai/registry/billing).
+
+:::
+
+## Simulate your machine with others
+
```

Comment:
This is moved down from the first section. And the info about live sim mode only being available in Simulate mode has been moved up from the tip further down.
```suggestion

You must be signed in to Stately Studio to initiate live simulation mode. 

There are two views in Stately Studio’s editor: **Design** mode and **Simulate** mode. **Live simulation** mode is available in **Simulate** mode.

```